### PR TITLE
Blogpost my switch from js to ruby

### PR DESCRIPTION
**Pull request by gersanco**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -190,6 +190,12 @@
   description: belongs to the epagesdevs content team.
   image: avatar_male.jpg
 
+- id: gsanemeterio
+  name: German
+  full_name: German
+  description: Here goes the description.
```
Comment (BirgitBader) [inline _data/authors.yml]: Suggestion: is a JavaScript Developer who is now into Ruby on Rails. He is fond of video games, as well as ...

> Would like make an addition here. What are you specifically fond of? Any technologies you also like?

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: My switch from JavaScript to Ruby
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: Suggestions: (I've put the search results analysis in brackets)

* How I became fond of Ruby on Rails (as a JavaScript developer) (50)
* Amazing how fast things can be done in Rails (69)
* When I learnt that Ruby on Rails is magick (65)

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+date: 2019-03-13
+header_image: public/my-switch-from-js-to-ruby.jpg
+header_position: center
+category: tech-stories
+tags: ["javascript", "ruby", "development"]
+authors: ["German"]
+about_authors: ["gsanemeterio"]
+---
+
+Hello, my name is German, I am currently part of ePages as software engineer with ruby as language, although before I worked with Angular.
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: As people read your name in the front matter already, I suggest to just state that you've started as a software developer in January.

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+When the opportunity of work in ePages came to me it cost me to take a decision but at the end I take the risk, as I like new challenges and to learn new things and I saw it was a great opportunity.
+
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: Add subtitle: Learning Rails

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+Now in ePages I started with the famous [Rails tutorial of Michael Hartl](https://www.railstutorial.org/){:target="_blank"}.
+During 3 weeks I was reading and trying it at the same time that I was doing the exercises.
+It's amazing how fast things can be done in rails!
+One finished the tutorial, I started to do pair programming with my teammate.
+He explained me the projects and cleared my dubs about ruby and, shortly after, I started with my own tasks, and apply that I learned in the tutorial.
+
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: Add subtitle: Discovering similarities and differences

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+While I was doing the tutorial I was seeing the similar things between rails and another languages with I work before or that I've seen such as Symfony.
+Due to my experience with other languages I could associate concepts of rails with other languages that I know.
+While I was doing the tutorial I was surprise with the easier that rails let to manage the database, thanks to ActiveRecord
+
+As each language there're differences and js and ruby couldn't been the exception, still today I'm thinking in how I do in javascript and then I search ruby's equivalent.
+
+Some of syntax differences with I found are:
+
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: You do list items here. Does each of the following examples refer to one of these items? Them better write the item and then have it followed by the example.

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+```js
+[0, 1, 2, 3, 4, 5].foreach((i) => {
+  console.log(`Value of local variable is ${i}`)
+})
+```
+
+- In ruby you don't have to declare variables.
+- Difference at other languages you don't have to type `return` to give back data, as always give back the result of the last line of code.
+
+When some friend told me about rails he mentioned one word `magick` today I finally understand why.
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: It's hard (at least for me) to understand lines 48 to 51. Can you shape that a bit?

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+- ActiveModel: I remember in my last work there a problem to make work Synfony with Oracle so when I saw ActiveRecord the first thing that I did was search in google information about rails and it looks there's not any problem to integrate rails and Oracle.
+- Trust: Rails with only type the name of some functions knows how to work.
+
+```ruby
+def index
+end
+```
+
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: Do you have some more examples?

File: _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+This is a little reflection from someone who has left of his comfort zone, I still have a lot to learn both JavaScript and Ruby, but that is another history.
```
Comment (BirgitBader) [inline _posts/2019/2019-03-13-my-switch-from-js-to-ruby.md]: before you end with this sentence, maybe you wanna give some insights on the first things you worked on, or what your next concrete steps are?

File: _data/authors.yml (hunk 1 above)
Comment (gersanco) [reply _data/authors.yml]: I need 

> Suggestion: is a JavaScript Developer who is now into Ruby on Rails. He is fond of video games, as well as ...
> 
> > Would like to make an addition here. What are you specifically fond of? Any technologies you also like?

In 3rd person?

Comment (BirgitBader) [reply _data/authors.yml]: Whatever you prefer. I just need some hints, and I'll shape it for you.

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+let result = sum(5, 6);
+```
+
+## It's magic(k)
+
+When some friend told me about Rails, he mentioned the word `magick`.
+Today I finally understand why.
+
+- ActiveModel: I remember that in my last job there was a problem to make Synfony work with Oracle. When I saw ActiveRecord the first thing that I did was search Google for information about Rails. It looks like there's no problem to integrate Rails and Oracle.
+- Trust: Rails with only type the name of some functions knows how to work.
```
Comment (BirgitBader) [inline _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: This is the only sentence remaining that I don't understand.

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+It's amazing how fast things can be done in Rails!
+When I was done with the tutorial, I started to do pair programming with my teammate right away.
+He explained me the projects and cleared away my doubts about Ruby.
+Shortly after that, I started with my own tasks, and applied what I learned in the tutorial.
+
+## Discovering similarities and differences
+
+While I was doing the tutorial I recognized the similar things between Rails and another languages with I worked with or that I've seen before, such as Symfony.
+Due to my experience with other languages I was able to associate concepts of Rails with other languages that I know.
+While I was doing the tutorial I was surprised how easy it was with Rails to manage a database, thanks to ActiveRecord.
```
Comment (unabris) [inline _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: Maybe put `ActiveRecord`?

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+
+```js
+function sum(num1, num2) {
+  return num1 + num2;
+}
+
+let result = sum(5, 6);
+```
+
+## It's magic(k)
```
Comment (unabris) [inline _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: I think it should me magic, without `k`

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+
+When some friend told me about Rails, he mentioned the word `magick`.
```
Comment (unabris) [inline _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: I think it should me magic, without `k`

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+Today I finally understand why.
+
+- ActiveModel: I remember that in my last job there was a problem to make Synfony work with Oracle. When I saw ActiveRecord the first thing that I did was search Google for information about Rails. It looks like there's no problem to integrate Rails and Oracle.
```
Comment (unabris) [inline _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: `ActiveRecord`, not `ActiveModel`

File: _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md (hunk 10 above)
Comment (unabris) [reply _posts/2019/2019-04-16-how-i-became-fond-of-ruby-on-rails-as-a-javascript-developer.md]: **What about something like this?**

- Conventions: Rails is one of those frameworks that follows the rule **Convention Over Configuration** resulting on (sometimes) writing empty methods like the following:

```ruby
class UsersController < ApplicationController
  def index
  end
end
```

Then Rails is automatically going to try to render a file named `index.*` inside of a folder called `users/`.

**Pull request merged**